Assets/Plugins/Android/Core/AndroidConfig.cs

```diff
@@ -25,6 +25,8 @@ namespace PlayEveryWare.EpicOnlineServices
     // Flags specifically for Android
     public class AndroidConfig : PlatformConfig
     {
+        public string GoogleLoginClientID;
```

Comment:
I don't know if we want to have these in the Android Config: for the time being, I would be okay if the values are in the sample code and need to be filled out by the person running the code.